client/src/app/user/user.service.ts

```diff
@@ -62,17 +65,24 @@ export class UserService {
   }
 
   saveUser(user: User): Observable<User> {
-    return this.authService.requireAuthOnce().pipe(switchMap(_ => this.db.userRef(user.id).set(user)), map(_ => user));
+    return this.authService.requireAuthOnce().pipe(
+      switchMap(_ => this.db.userRef(user.id).set(user)),
+      map(_ => user)
+    );
   }
 
   updateUser(user: User): Observable<User> {
-    return this.authService
-      .requireAuthOnce()
-      .pipe(switchMap(_ => this.db.userRef(user.id).update(user)), map(_ => user));
+    return this.authService.requireAuthOnce().pipe(
+      switchMap(_ => this.db.userRef(user.id).update(user)),
+      map(_ => user)
```

Comment:
Maybe I'm missing something here but as the output of the map is always `user` couldn't the be a `mapTo`

Reply:
Yes, you're absolutely correct! But this was not introduced by any of the commits and has nothing to do with this particular feature. This can definitely be refactored in a future commit / PR.